x-pack/plugins/reporting/public/management/components/report_info_flyout.tsx

```diff
@@ -38,7 +38,9 @@ interface Props {
 
 export const ReportInfoFlyout: FunctionComponent<Props> = ({ config, onClose, job }) => {
   const isMounted = useMountedState();
+
   const { apiClient } = useInternalApiClient();
+  const isServerless = apiClient.getIsServerless();
```

Comment:
The overall changes in this PR can be simplified, since there already is an indication of whether the deployment is serverless in the `config` object that is in scope here